data/events/2020-toronto.yml

```diff
@@ -18,7 +18,7 @@ cfp_date_start:  2019-12-01T00:00:00-05:00 # start accepting talk proposals.
 cfp_date_end:  2020-01-31T23:59:59-05:00 # close your call for proposals.
 cfp_date_announce:  2020-02-29T00:00:00-05:00 # inform proposers of status
 
-cfp_link: "https://www.papercall.io/dodto2020" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+# cfp_link: "https://www.papercall.io/dodto2020" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
```

Comment:
You should change the value to "" instead of commenting out the line. If you look back through the file history you'll see that what the value was before I setup the CFP.

Reply:
Good call! Will update that!